axelrod/_strategy_utils.py

```diff
@@ -61,29 +74,28 @@ def look_ahead(player_1, player_2, game, rounds=10):
     for strategy in strategies:
         # Instead of a deepcopy, create a new opponent and play out the history
         opponent_ = player_2.clone()
-        player_ = Cycler(strategy) # Either cooperator or defector
+        player_ = Cycler(strategy)  # Either cooperator or defector
         for h1 in player_1.history:
             limited_simulate_play(player_, opponent_, h1)
 
-        round_robin = RoundRobin(players=[player_, opponent_], game=game,
-                                 turns=rounds)
         simulate_match(player_, opponent_, strategy, rounds)
-        results.append(round_robin._calculate_scores(player_, opponent_)[0])
+        results.append(calculate_scores(player_, opponent_, game))
 
     return strategies[results.index(max(results))]
 
 
 class Memoized(object):
-   """Decorator. Caches a function's return value each time it is called.
-   If called later with the same arguments, the cached value is returned
-   (not reevaluated). From:
-   https://wiki.python.org/moin/PythonDecoratorLibrary#Memoize
-   """
-   def __init__(self, func):
+    """Decorator. Caches a function's return value each time it is called.
+    If called later with the same arguments, the cached value is returned
+    (not reevaluated). From:
+    https://wiki.python.org/moin/PythonDecoratorLibrary#Memoize
+    """
+
+    def __init__(self, func):
         self.func = func
         self.cache = {}
 
-   def __call__(self, *args):
+    def __call__(self, *args):
```

Comment:
Sorry about the spacing on these! That's what I get for copying from the python wiki.